client/app/lib/views/credentiallist/accountcredentiallist.coffee

```diff
@@ -24,14 +24,22 @@ module.exports = class AccountCredentialList extends KodingListView
   showCredential: (options = {}) ->
 
     { credential, cred } = options
+    
+    content = "<p><pre><code>#{cred}</code></pre></p>"
```

Comment:
by putting the `tagName: 'p'` you can get rid of the `p` wrapper in `content`, pls see the code above.